src/core/openthread.cpp

```diff
@@ -1636,6 +1636,21 @@ ThreadError otSetActiveDataset(otInstance *aInstance, const otOperationalDataset
     return error;
 }
 
+bool otIsNodeCommissioned(otInstance *aInstance)
+{
+    otOperationalDataset aDataset;
+
+    otGetActiveDataset(aInstance, &aDataset);
```

Comment:
As far as I can tell, the Windows build is failing because the [Windows implementation](https://github.com/openthread/openthread/blob/master/examples/drivers/windows/otApi/otApi.cpp#L2100) is conflicting with the [OpenThread implementation](https://github.com/openthread/openthread/blob/master/src/core/openthread.cpp#L1615).

One fix could be to make the C++ call directly:
```
aInstance->mThreadNetif.GetActiveDataset().GetLocal().Get(dataset);
```

@nibanks, any thoughts?